src/components/organisms/plans/plans.html

```diff
@@ -0,0 +1,55 @@
+<div class="organisms-plans">
+  <div class="message-wrapper">
+    <h2>ライフスタイルに合わせて選べる多彩なプラン。</h2>
+    <p>
+      平日のみのプラン、仕事帰りをコアタイムとした夕方以降が中心のプラン、休日をメインとしたプランなど、多彩なプランの中からそれぞれのお客様のライフスタイルに合わせてお選びいただけます。ビジター料金でのご利用も可能ですので、月額会員登録の前に体験をしたいという方もぜひご利用ください。
+    </p>
+  </div>
+  <div class="plan-table-wrapper">
+    <ul class="plan">
+      <li class="plan-text">デイタイムプラン</li>
+      <li class="price-outline">
+        <div class="plan-price">
+          <span class="yen">¥</span>
+          <span class="five-thouthand">5,000</span>
+        </div>
+        <p class="monthly_amount">月額</p>
+      </li>
+      <li class="basic_info">入会金0円</li>
+      <li class="basic_info">平日:9時～17時</li>
+      <li class="basic_info">土日祝:利用なし</li>
+      <li class="basic_info">コアタイムでの利用に最適なプラン</li>
+    </ul>
+    <ul class="plan">
+      <li class="plan-text">スタンダードプラン</li>
+      <li class="price-outline">
+        <div class="plan-price">
+          <span class="yen">¥</span>
+          <span class="five-thouthand">15,000</span>
+        </div>
+        <p class="monthly_amount">月額</p>
+      </li>
+      <li class="basic_info">入会金0円</li>
+      <li class="basic_info">平日:9時～17時</li>
+      <li class="basic_info">土日祝:利用なし</li>
+      <li class="basic_info">コアタイムでの利用に最適なプラン</li>
+    </ul>
+    <ul class="plan">
+      <li class="plan-text">プレミアムプラン</li>
+      <li class="price-outline">
+        <div class="plan-price">
+          <span class="yen">¥</span>
+          <span class="five-thouthand">20,000</span>
+        </div>
+        <p class="monthly_amount">月額</p>
+      </li>
+      <li class="basic_info">入会金5000円</li>
+      <li class="basic_info">平日：営業時間全て</li>
+      <li class="basic_info">土日祝：営業時間全て</li>
+      <li class="basic_info">フルに使いたい方のためのプレミアムなプラン</li>
+    </ul>
+  </div>
+  <div class="plan-fee-button">
+    <a href="#" class="button">プラン料金</a>
+  </div>

```

Comment:
```suggestion
    <a href="#" class="button">プラン料金</a>
```
こちらで問題ないと思います。
下の課題ページに少し書いてありますが、scssの入れ子はcssに変換した際に 子孫セレクタ の関係になります。
scssを入れ子にしておけば `organisms-plan` の中の `.button` にのみスタイル適用されることになり、他の `.button` に影響を与えないのでわざわざ別のクラスをセレクタに使わなくても大丈夫という感じです。
参考： https://kita-note.com/css-ketugoushi
課題ページ： https://shinonome.io/frontend-basic/lastAssignment/storybook_sass/